docs/examples/tree-v2/disabled.vue

```diff
@@ -6,6 +6,7 @@ interface Tree {
   id: string
   label: string
   children?: Tree[]
+  disabled?: boolean
```

Comment:
There is no need to set optional here, according to the code you added, each node has this property.